charger/easee.go

```diff
@@ -320,6 +323,8 @@ func (c *Easee) CommandResponse(i json.RawMessage) {
 		return
 	}
 	c.log.TRACE.Printf("CommandResponse %s: %+v", res.SerialNumber, res)
+
+	c.respChan <- res
```

Comment:
Blockt das nicht? Oder warten wir jetzt sicher immer auf ein Ergebnis?

Reply:
Ja, guter Punkt.
Das blockt und eigentlich warten wir immer auf ein Ergebnis. "Normalerweise" (TM) sollte auch immer ein receiver am andere Ende stehen. Aber wer weiß was wir hier für Timings sehen. Wenn wir beim Empfänger in das 10s Timeout laufen, ist der Receiver weg. Dann würde das hier hängen. Es macht auch keinen Sinn mehr in der Situation noch die Nachricht in den Channel zu senden. Ich baue einen Default ein, der die verworfene Nachricht logged und das wars dann.

Reply:
Bitte nicht noch mehr logging…